INSTALLING.rst

```diff
@@ -180,19 +180,19 @@ Run ``python3 install-prereq-headers.py -h`` to see a list of the command line o
 Compile HOOMD-blue
 ------------------
 
-Configure::
-
-    $ cd /path/to/hoomd-blue
-    $ cmake -B build
-    $ cd build
-
 .. warning::
```

Comment:
The section *Configure a virtual environment* includes instructions to set `CMAKE_PREFIX_PATH`. If that is too far from the `cmake` command that requires, I think we should move that here and delete this warning block.

Reply:
Thanks for pointing that out!